src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -3330,20 +3577,18 @@ TEST_CASE("state archival operation errors", "[tx][soroban][archival]")
                                             1'000, DEFAULT_TEST_RESOURCE_FEE);
             REQUIRE(!test.isTxValid(tx));
         }
+
+        // Read byte limits no longer apply to extension, since the footprint is
+        // always live soroban state
         SECTION("exceeded readBytes")
```

Comment:
I think this needs to be TEST_CASE_VERSIONS as well.

Reply:
Fixed